include/dsVideoPort.h

```diff
@@ -284,8 +286,9 @@ typedef void (*dsVideoFormatUpdateCB_t)(dsHDRStandard_t videoFormat);
 /**
  * @brief Checks whether a video port is active or not.
  * 
- * This function is used to indicate whether a video port is active or not. A HDMI output port is active if it is connected 
- * to the active port of sink device. 
+ * For Source devices, this function is used to indicate whether a video port is active or not. A HDMI output port is active if it is connected
+ * to the active port of sink device.
+ * For sink devices, this function shall return video port connection status as active and returns dsERR_NONE always.
```

Comment:
The statement still lacks clarity.  Do we need to give reference to dsVIDEOPORT_TYPE_INTERNAL as Sink's own display?

Reply:
Updated